### ISSUE TITLE
500 error when previewing a fragment with  embedded iframe

### ISSUE THREAD

**Issue by sgauruseu** [enonic/app-contentstudio#3979]
1. Open Features, insert `embed iframe` in the existing `hello-word` text component 
2. insert this URL   `<iframe src='http://www.enonic.com'> enonic</iframe>`   in the configuration text area
![image](https://user-images.githubusercontent.com/3728712/146226339-058d841d-25c1-4ad8-a15a-f307961e5be4.png)

2. Save the text component  as fragment
![image](https://user-images.githubusercontent.com/3728712/146226938-a57cba63-b88a-4650-a9d0-931d05464295.png)

**BUG - error in the page preview , 500 error in the browser console**

```
311c2635-65f0-4563-94b2-374b8251d662:1 Failed to load resource: the server responded with a status of 500 (Server Error)
````

**Comment by alansemenov** [comment]
We should not have inline preview for embed macro at all - preview should either open in a separate browser tab or be totally disabled.

This should be solved before we release CS 4.1.0 with enabled CSP.

**Comment by alansemenov** [comment]
This works now (iframe itself is shown as blocked content which is correct).

![image](https://user-images.githubusercontent.com/11612490/162432062-b9c30161-eefe-4e63-8650-98f40c744d91.png)
